src/lib.rs

```diff
@@ -160,10 +160,44 @@
 // False positives when block expressions are used inside an assertion.
 #![allow(panic_params)]
 
+#![no_std]
+#![cfg_attr(feature = "nightly", feature(alloc))]
```

Comment:
Is there a reason for this to use a `nightly` feature, instead of `not(feature = "std")` like we do elsewhere? It'd be nice if `cargo build --no-default-features` could work.

Reply:
Yes. See Cargo.toml:
```
nightly = ["fallible-iterator/alloc"]
```

Reply:
Ok, this is probably nitpicky, but I just want to make sure I understand if you don't mind:
- do we need to use `fallible-iterator/alloc` in gimli?
- why don't the uses of the alloc crate depend on the `nightly` feature?

Reply:
> do we need to use fallible-iterator/alloc in gimli?

Yes. Otherwise, it cannot be tested without the `std` feature, I believe.